### PR TITLE
Task/confluent docs

### PR DESCRIPTION
**Pull request by odacremolbap**
- Improves slack and elastic docs by removing technical details and using brokers as sinks for the examples
- Add confluent docs

### CODE REVIEW

File: docs/targets/sendgrid.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+- [SendGrid event target](#sendgrid-event-target)
+  - [Contents](#contents)
+  - [Prerequisites](#prerequisites)
+  - [Creating SendGrid Target](#creating-sendgrid-target)
+  - [Expected CloudEvents](#expected-cloudevents)
+
+## Prerequisites
+
+A SendGrid account API token
```
Comment (cab105) [inline docs/targets/sendgrid.md:15]: Would we want to provide a link on how to obtain it?

File: docs/targets/twilio.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+- [Twilio event target](#twilio-event-target)
+  - [Contents](#contents)
+  - [Prerequisites](#prerequisites)
+  - [Creating an Twilio Target](#creating-an-twilio-target)
+  - [Expected CloudEvents](#expected-cloudevents)
+
+## Prerequisites
+
+A TWilio configured account is required to run this target:
```
Comment (cab105) [inline docs/targets/twilio.md:15]: Similar to Sendgrid. Do we want to provide a link on how to obtain the token?  Also, think you meant `Twilio`

**Comment by odacremolbap** [comment]
@cab105 I think fleshing out this target info is a good idea, in some cases it is just adapting some of the info from the targets repo.

Can we do that in a follow up issue?
Or do you mean just a link?

**Comment by cab105** [comment]
I was thinking just a link. Anything more than that would be doing the vendor's job for them.

**Comment by odacremolbap** [comment]
> I was thinking just a link. Anything more than that would be doing the vendor's job for them.

@cab105 thanks for your review

comments addressed.

**Pull request merged**